articles/getting-started/the-basics.md

```diff
@@ -13,25 +13,23 @@ useCase:
+Often, the biggest barrier to learning new things, especially in the tech industry, is terminology. The words that are used to describe things can cause problems when you try to understand new concepts. This document explains some of the basic terminology we use here at Auth0, and maps these terms to concepts you are already familiar with. We also have a handy [glossary](/glossary).
 
 ::: panel TL;DR
-This article uses an example to introduce some core concepts of Auth0: **accounts**, **tenants**, **domains**, **applications**, and **connections**. If you already know what these are (in the context of Auth0) you can safely skip reading it.
+This article uses an example to introduce some core concepts of Auth0: **accounts**, **tenants**, **domains**, **applications**, and **connections**. If you already know what these are in the context of Auth0, you can safely skip reading it.
 
 We will use a very simple example: A company named `Example-Co` wants to use Auth0 for authentication. They have a web app and a mobile app, and they want their users to be able to log in with username/password, Google, or GitHub.
 :::
 
 ## Account and tenants
 
-If you haven't already [signed up](https://auth0.com/signup) for an Auth0 **account**, do so (it's free). You can either use username/password or log in with a social provider (GitHub, Google, or Microsoft).
+If you haven't already [signed up](https://auth0.com/signup) for an Auth0 **account**, do so (it's free). You can either use username and password or log in with a social provider (such as Facebook, Google, or Apple).
 
-Once you create your account you will be asked to create a **Tenant**. This is a **logical isolation unit**.
+Once you create your account you will be asked to create a tenant. In Auth0, a tenant is a **logical isolation unit**. The term is borrowed from *software multitenancy* and it refers to an architecture where a single instance of the software serves multiple tenants. No tenant can access the instance of another tenant, even though the software might be running on the same machine (hence the logical isolation).
 
-The term is borrowed from "software multitenancy". This refers to an architecture where a single instance of the software serves multiple tenants. No tenant can access the instance of another tenant, even though the software might be running on the same machine (hence the logical isolation).
+Tenant characteristics:
 
-Some characteristics:
-
-- The tenant name has to be unique (we will see in the next paragraph that it is used to create your own personal domain).
+- The tenant name has to be unique. (See in the next paragraph that it is used to create your own personal domain).
```

Comment:
> (See in the next paragraph that it is used to create your own personal domain).

The tenant name has to be unique. It will be used to create your personal domain.

Reply:
Updated text